test/mocha/20-validate-config.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+/* globals should */
+
+'use strict';
+
+const bedrock = require('bedrock');
+const brValidator = require('bedrock-ledger-validator-equihash');
+
+const mockData = require('./mock.data');
+
+describe('validateConfiguration API', () => {
+  it('validates a proper config', done => {
+    const testConfig = mockData.ledgers.alpha.config.input[0].eventValidator[0];
+    brValidator.validateConfiguration(testConfig, err => {
+      should.not.exist(err);
```

Comment:
I'd prefer `if(err) return done(err);` for these situations, better to work with when developing and you're getting errors in tests.